vignettes/tcplfit2-vignette.Rmd

```diff
@@ -348,7 +348,7 @@ For this example, the normalized responses (`resp`) are calculated as a percent
 $$
 \% \space control = \frac{cval - bval}{pval - bval}
 $$
-The table below provides a few methods for calculating `bval` and `pval` in tcpl. For more on the data normalization step, refer to the [Data Normalization](https://cran.r-project.org/web/packages/tcpl/vignettes/Data_processing.html#data-normalization) section of `tcpl`'s Data Processing Vignette. 
+The table below provides a few methods for calculating `bval` and `pval` in `tcpl`. For more on the data normalization step, refer to the [Data Normalization](https://cran.r-project.org/web/packages/tcpl/vignettes/Data_processing.html#data-normalization) section of `tcpl`'s Data Processing Vignette. 
```

Comment:
vignette link will be out of date with next CRAN release 

For more on the data normalization step, refer to the data processing section of the `tcpl`'s vignette 
 https://cran.r-project.org/web/packages/tcpl/index.html

Reply:
@madison-feshuk, do we have OR are we able to anticipate the actual vignette link for the next CRAN release?  This one will just take an individual to the landing page on CRAN for tcpl, which is fine.  However, it would be ideal to go directly to the vignette and relevant section.